content/08-cardano-sidechains/03-example-evm-sidechain/06-transferring-tokens.mdx

```diff
@@ -0,0 +1,222 @@
+Input the amount of `SC_Token`, in `wei`, you want to lock: eg, `1000000000` Wei == 1 `SC_Token`. Your Remix should look similar to this: ![Remix lock token](../../images/remix-lock-token.png).
+- Select the `lock` button to initiate the transaction and follow the MetaMask prompts to confirm and sign the transaction.
+
+*Note: If the transaction appears to be stuck, 'pending' status, or MetaMask shows an invalid balance, try [resetting MetaMask](https://metamask.zendesk.com/hc/en-us/articles/360015488891-How-to-reset-an-account) and submit the transaction again.*
+
+- To get the transaction hash or verify that the transaction was successful, you can check MetaMask activity or the Remix console: ![Remix console](../../images/remix-console.png).
+  - Alternatively, you can invoke the `eth_gettransactionReceipt` API method. You can request this until the receipt is returned or transaction times out:
+  ```shell
+  curl -L -X POST -H 'Content-type:application/json' -d '{
+  "jsonrpc": "2.0",
+  "method": "eth_getTransactionReceipt",
+  "params": [INSERT_RAW_TX],
+  "id": 1
+  }' https://faucet.sidechain.evmtestnet.iohkdev.io/ | jq
+  ```
+
+#### Step 04 - Wait for transaction to be processed and obtain MerkleProof
+
+- Get the current sidechain status by invoking `sidechain_getStatus`:
+
+```shell
+curl -L -X POST -H 'Content-type:application/json' -d '{
+   "jsonrpc": "2.0",
+   "method": "sidechain_getStatus",
+   "params": [],
+   "id": 1
+   }' https://faucet.sidechain.evmtestnet.iohkdev.io/ | jq -r
+```
+
+This shows on which epoch and phase the `lock()` transaction happened. If it happened during epoch `N` on epochPhase `regular` you can obtain the Merkle proof on epoch `N` since epochPhase changes to `handover`. If it happened during epoch `N` on epochPhase `closedTransactionBatch` or `handover`, you can obtain the Merkle proof on epoch `N+1` since epochPhase changes to `handover`.
+
+- Get transaction index for the epoch the lock transaction will be processed by invoking the `sidechain_getOutgoingTransactions` API method:
+
+```
+curl -L -X POST -H 'Content-type:application/json' -d '{
+  "jsonrpc": "2.0",
+  "method": "sidechain_getOutgoingTransactions",
+  "params": [<N or N+1>],
+  "id": 1
+  }' https://faucet.sidechain.evmtestnet.iohkdev.io/ | jq
+```
+
+Example output:
+
+```json
+{
+  "proof": {
+    "bytes": "0xd8799fd8799f0002581d606e9d4f6a3f900f7b510b27f29d8e79c550686ce093946b23b3d1828ed87a80ff80ff",
+    "info": {
+      "transaction": {
+        "value": "0x2",
+        "recipient": "0x606e9d4f6a3f900f7b510b27f29d8e79c550686ce093946b23b3d1828e",
+        "txIndex": 0
+      },
+      "merkleRootHash": "0x96e0bffb3786607057ddffa5bc3a92c5b6669109c12b5222da374cd99a815b4e"
+    }
+  }
+}
+```
+
+To obtain the `txIndex` matching the recipient and amount used to lock tokens, proceed as follows.
```

Comment:
do we want to obtain `txIndex` or `merkleProof`?

Reply:
committed, merkleproof